go.mod

```diff
@@ -1,18 +1,18 @@
-module github.com/open-cluster-management/governance-policy-template-sync
+module github.com/stolostron/governance-policy-template-sync
 
 go 1.14
 
 require (
 	github.com/onsi/ginkgo v1.14.1
 	github.com/onsi/gomega v1.10.2
-	github.com/open-cluster-management/governance-policy-propagator v0.0.0-20200602150427-d0f4af8aba9d
-	github.com/operator-framework/operator-sdk v0.17.0
+	github.com/operator-framework/operator-sdk v0.18.1
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.17.4
-	k8s.io/apimachinery v0.17.4
+	github.com/stolostron/governance-policy-propagator v0.0.0-20220112203818-b0bc3438ecd7
+	k8s.io/api v0.18.3
+	k8s.io/apimachinery v0.18.3
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v1.0.0
-	sigs.k8s.io/controller-runtime v0.5.2
+	sigs.k8s.io/controller-runtime v0.6.0
 )
 
 replace (
```

Comment:
missing replace. old api path still showing up in https://github.com/stolostron/governance-policy-template-sync/blob/b018fe140a06b7e76f83cf6f7cb2ffde00ece322/go.sum#L625

Reply:
Shoot--I didn't know it wouldn't replace it here. Thanks!